### ISSUE TITLE
Why use adam optimizer ,train loss only at the beginning of each epoch it drops soon and then slows down？

### ISSUE THREAD

**Issue by xiaomaxiao** [keras-team/keras#8289]
I found use adam optimizer ,train loss  only at the beginning of each epoch it drops soon and then slows down。

like this：
epoch 1/100： loss 70.0 -> 60.0 ->55. -> ...... ->40.0  ->39.95 ->39.90->39.89->.....
epoch 2/100： loss 30 -> 25 ->20 -> ...... ->18.0  ->17.9  ->.....->17.75 ->17.73
epoch 3/100： loss 10 -> 8 ->7 -> ...... ->6  ->6.9  ->.....
...
